### ISSUE TITLE
Extension stops working with webstorm 2023.3 - Please update this extension

### ISSUE THREAD

**Issue by tausifcreates** [albertoventurini/graphdb-intellij-plugin#87]
Message: Requires IDE build 232.* or older

**Comment by lavazais-sylvain** [comment]
![image](https://github.com/albertoventurini/graphdb-intellij-plugin/assets/134276360/a9e4467e-3f6f-4275-a73a-9fe29b4ac9d3)

IntelliJ about : 
```
IntelliJ IDEA 2023.3 (Ultimate Edition)
Build #IU-233.11799.241, built on December 2, 2023
Runtime version: 17.0.9+7-b1087.7 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 14.1.2
GC: G1 Young Generation, G1 Old Generation
Memory: 4096M
Cores: 10
Metal Rendering is ON
Kotlin: 233.11799.241-IJ
```

**Comment by vexos-io** [comment]
Yes. There is only one plugin for cypher/neo4j, so the update is Make-Sun-Rise-Again.

**Comment by oyvindwe** [comment]
As mention previously in https://github.com/albertoventurini/graphdb-intellij-plugin/issues/75#issuecomment-1653177974, I think it's better to remove the upper bound. Then the plugin will function until a code change is required, instead of failing to load with every minor(?) update of IntelliJ.

And thank you for maintaining this plugin - I find it helpful on a regular basis!

**Comment by albertoventurini** [comment]
Thank you for opening this issue, and apologies for not acting on this sooner. I will create a patch release for this soon.

@oyvindwe, I acknowledge your comment. Case in point, updating to IDEA 2023.3 indeed requires some (minor) code changes, which means that removing the upper bound would not have worked in this case.

That said, I am willing to relax the upper bound for now and see how it goes. Due to personal life, I don't have much bandwidth to work on this plugin at the moment. Therefore, I am OK with relaxing the upper bound for now. If the plugin were to break after an upgrade, then it can be fixed at that point.

**Comment by tausifcreates** [comment]
> Thank you for opening this issue, and apologies for not acting on this sooner. I will create a patch release for this soon.
> 
> @oyvindwe, I acknowledge your comment. Case in point, updating to IDEA 2023.3 

Thank you so much for addressing this!

**Comment by albertoventurini** [comment]
I've just published release 0.8.0 which adds support for 2023.3: https://github.com/albertoventurini/graphdb-intellij-plugin/releases/tag/v0.8.0. It can already be installed manually in IntelliJ and other IDEs.

I will submit it to JetBrains soon.

**Comment by albertoventurini** [comment]
The new release is now available on JetBrains, thus closing this issue
